mmv1/products/compute/VpnGateway.yaml

```diff
@@ -126,6 +132,17 @@ properties:
       The network this VPN gateway is accepting traffic for.
     required: true
     immutable: true
+  - !ruby/object:Api::Type::Enum
+    name: 'stack_type'
```

Comment:
You need to use the API property name here not the terraform name. See other properties.